panda/src/putil/loaderOptions.I

```diff
@@ -18,6 +18,9 @@ constexpr LoaderOptions::
 LoaderOptions(int flags, int texture_flags) :
   _flags(flags),
   _texture_flags(texture_flags),
+  _texture_format(0),
+  _texture_compress(0),
+  _texture_quality(1),
```

Comment:
I would prefer if texture_quality defaulted to 0 (`QL_default`), which tells Panda to use whatever is specified in the configuration file.